frontend/package.json

```diff
@@ -11,13 +11,18 @@
         "format": "prettier --write \"./**/*.{js,jsx,mjs,cjs,ts,tsx,json}\""
     },
     "dependencies": {
+        "@emotion/react": "^11.13.3",
```

Comment:
do we need these packages? seems like might be redundant because we already have existing frameworks?

Reply:
These packages are used for MUI Icons, but if you think it shouldnt be used then i will try find svgs to replace the icon in the toast instead

Reply:
Yes, sounds good +1